src/test/java/io/aiven/kafka/connect/http/config/HttpSinkConfigTest.java

```diff
@@ -319,6 +321,38 @@ void headerContentType() {
         assertEquals("application/json", config.headerContentType());
     }
 
+    @Test
```

Comment:
Could you add another test where we successfully check that one can add additional headers that only differ in the case?
For example:
```
"http.headers.additional", "test:value,TEST:valueUpperCase"

Reply:
From what i understood header fields are case-insensitive, what would be the use case for allowing "duplicate" case sensitive headers?

Reply:
Sorry, my bad. What I mean here is to check for this corner case where we can, indeed, provide headers whose key only differs by their case.
The current implementation already supports it. It's just to make sure this case is covered and any future refactoring takes this into consideration.

For example if we provide these: `"http.headers.additional", "test:value,TEST:valueUpperCase"` we should check that 

```java
assertEquals(2, additionalHeaders.size());
assertEquals("value", additionalHeaders.get("test"));
assertEquals("valueUpperCase", additionalHeaders.get("TEST"));
```

Reply:
I understood, what I am asking is whether it makes sense for case sensitive field names to be supported or should we also restrict this? (from what I understood header field names are case-insensitive, as specified in [RFC](https://datatracker.ietf.org/doc/html/rfc7230#section-3.2))

Reply:
Oh, you are completely right! I missed this part of the specification.
Then we should add this check in the validator, "Keys should be unique in a case insensitive way".

Reply:
Added duplicate keys check in validator that takes into account the header field case